src/main/java/com/eppo/sdk/dto/AssignmentLogData.java

```diff
@@ -29,7 +32,7 @@ public AssignmentLogData(
         this.featureFlag = featureFlag;
         this.allocation = allocation;
         this.variationValue = variationValue;
-        this.variation = variationValue.stringValue();
+        this.variation = variationName;
```

Comment:
Comment (blocking): did we accidentally blow-away the `variation`?

I think we want this:

```suggestion
        this.variation = variationValue.stringValue();
        this.variationName = variationName;
```

Reply:
We have the `variationValue`, which is an EppoValue (typed).

Did we still want the `variation` to be a string representation of the eppo value?

Reply:
Spoke offline. `this.variation = variationValue.stringValue()` was part of Eppo's initial implementation. To maintain backwards compatibility we're going to leave that property as-is.

As such, I'm going to add a `variationName` property to capture the name of the variation.